app/models/deletion.rb

```diff
@@ -57,11 +57,13 @@ def expire_cache
   def remove_from_index
     version.update!(indexed: false, yanked_at: Time.now.utc)
     reindex
+    Rstuf::RemoveJob.perform_later(version: version)
```

Comment:
```suggestion
    Rstuf::RemoveJob.perform_later(version:)
```

Reply:
I'm not big fan of this syntax, but it seems already used in this file. I'll update, thanks for suggestion.